draft-xyz-rats-psa-endorsements.md

```diff
@@ -53,22 +53,29 @@ informative:
 
 --- abstract
 
-This memo describes a CoMID profile to convey PSA Endorsements.
-
-PSA Endorsements comprise reference values, cryptographic key material and
+PSA Endorsements comprises reference values, cryptographic key material and
 certification status information that a Verifier needs in order to appraise PSA
 Evidence.
 
+This memo describes the data formats used to convey PSA Endorsements to an Attestation Verifier.
+The format presented in this document is a PSA specific profile of CoRIM/CoMID based information model presented in
+Trusted Computing Group (TCG) Endorsement Architecture.
```

Comment:
Just a note: I think all these references to TCG are going to be made replaced to refs to draft-birkholz-corim as soon as it gets published.

Reply:
Correct, needs references to be added once published!